tools/dependencies-report/src/main/resources/licenseMapping.csv

```diff
@@ -102,7 +102,7 @@ dependency,dependencyUrl,licenseOverride
 "manticore:0.6.4","https://github.com/cheald/manticore/",MIT
 "march_hare:4.1.1","https://github.com/ruby-amqp/march_hare/",MIT
 "memoizable:0.4.2","https://github.com/dkubb/memoizable/",MIT
-"method_source:1.0.0","https://github.com/banister/method_source/",MIT
+"method_source:0.9.2","https://github.com/banister/method_source/",MIT
```

Comment:
@roaksoax There is no actual downgrade - the fix that @jsvd made was to accurately report on the dependencies referenced in the lock file, rather than the latest versions available at the time the report was run, which led to having to set newer versions in this mapping file to fix the report.

Reply:
A previous bug caused us to use the latest release of a gem instead of the one in the lock file. We bumped the license versions so the 7.6 license checker would pass. Now that we respect the lock file we can reset the versions used there.